tests/foreman/sys/test_hot_backup.py

```diff
@@ -0,0 +1,156 @@
+# -*- encoding: utf-8 -*-
+"""Test class for ``katello-backup``
+
+@Requirement: HotBackup
+
+@CaseAutomation: notautomated
+
+@CaseLevel: Acceptance
+
+@CaseComponent: Backup
+
+@TestType: System
+
+@CaseImportance: High
+
+@Upstream: No
+"""
+
+from robottelo.decorators import stubbed
+from robottelo.test import TestCase
+
+
+class HotBackupTestCase(TestCase):
+    """Implements ``katello-backup --online`` tests"""
+
+    @classmethod
+    def setUpClass(cls):
+        super(HotBackupTestCase, cls).setUpClass()
+
+    @stubbed()
+    def test_positive_directory(self):
+        """katello-backup with existing directory
+
+        @id: 3f8285d5-c68b-4d84-8568-bcd7e4f7f19e
+
+        @Steps:
+            1. Populate Satellite system.
+            2. Run katello-backup with existing directory
+            3. Restore from backup
+
+        @Assert: The backup and restore succeed.
```

Comment:
Are we talking about asserting that the return code is zero? Any other checks need to be performed, such as checking that the content used to populate the system is still around?